### ISSUE TITLE
Success color and text

### ISSUE THREAD

**Issue by gfellerph** [swisspost/design-system#2227]
The success color changed: https://www.figma.com/file/ojCcgC5Zd12eUSzq6V5m24/Foundations-Post?type=design&node-id=26-111&mode=design

The text on success background should be white. According to the currently valid contrast algorithm, black has a slightly better contrast than white while both pass the check. WCAG 3.0 will use APCA as contrast algorithm and with this change, black on success will be a fail and only white will be valid. Introducing APCA now would be too big of a change so we should make an exception for white on success.
